constraints/miden-vm/memory.air

```diff
@@ -0,0 +1,101 @@
+mod MemoryChipletAir
+
+### Helper functions/evaluators ###################################################################
+
+fn delta(v: scalar) -> scalar:
+    return v' - v
+
+
+# Enforces that columns must be zero.
+ev is_zero(main: [value[4]]):
+    enf v = 0 for v in value
+
+
+# Enforces that values in the columns in the current row must be equal to the values in the next 
+# row.
+ev equal_seq_lines(main: [value[4]]):
```

Comment:
Question: Do you think a more appropriate name here would be something like `is_unchanged`?

nit: Should this be values instead of value or is the idea to consider a word a single value? If so, should we rename this to word instead? Wdyt?